pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -113,7 +113,7 @@ func (r *SystemReconciler) Reconcile() (reconcile.Result, error) {
 		sidekiqDCMutators = append(sidekiqDCMutators, reconcilers.DeploymentConfigReplicasMutator)
 	}
 
-	err = r.ReconcileDeploymentConfig(system.SidekiqDeploymentConfig(), reconcilers.DeploymentConfigMutator(sidekiqDCMutators...))
+	err = r.ReconcileDeploymentConfig(system.SidekiqDeploymentConfig(*r.apiManager.Spec.PriorityClassName), reconcilers.DeploymentConfigMutator(sidekiqDCMutators...))
```

Comment:
same q as above

Reply:
Thank you for comment, please see reply above